lib/spack/spack/test/url_fetch.py

```diff
@@ -173,6 +175,22 @@ def test_unknown_hash(checksum_type):
         crypto.Checker('a')
 
 
+def test_url_with_status_bar(tmpdir, mock_archive, monkeypatch):
+    """Ensure a fetch with status bar option."""
```

Comment:
It's not clear to me how does this test ensure that the status bar option is displayed.

Reply:
I will change the comment to be clearer since it mocks the `tty` functions to cover the statements that were causing patch coverage to be shy of the required level.

Reply:
I ended up changing the comment *and* adding an assert on the output of the test.